integration-tests/integration.test.js

```diff
@@ -1,7 +1,13 @@
 const { getByText, fireEvent, waitFor } = require('@testing-library/dom')
 const { startApp } = require('./test-app')
+const { startApp: startBrokenApp } = require('./broken-app')
 
 describe('Tram-One', () => {
+	beforeEach(() => {
+		// clean up any tram-one properties between tests
+		window['tram-space'] = undefined
+	})
```

Comment:
I need to do this for the hook test (because that's how we know that a hook was called in the wrong context.